cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinBiobank.java

```diff
@@ -0,0 +1,189 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings("PMD.ExcessiveParameterList")
+public final class CristinBiobank {
+    //Here be more params
+    private static final String CRISTIN_BIOBANK_ID = "cristin_biobank_id";
+    private static final String CRISTIN_NAME_FIELD = "name";
+
+    private static final String CRISTIN_BIOBANK_COORDINATING_INSTITUTION = "institution";
+
+    private static final String CRISTIN_BIOBANK_LAST_MODIFIED = "last_modified";
```

Comment:
This is also a good way to define name for a constant. The important thing when giving names is we understand easily which constant map to each field.